contracts/MyNFT.sol

```diff
@@ -12,20 +12,27 @@ contract MyNFT is ERC721URIStorage, Ownable {
     using Counters for Counters.Counter;
     Counters.Counter private _tokenIds;
 
+    Counters.Counter private randNonce;
+
+    uint256 private lastRaceResult = 0;
```

Comment:
not sure about this, do we really need it?

Reply:
to be able to return in getRaceResult.... :(